### PR TITLE
Styling changes in Story Board

### PR DESCRIPTION
**Pull request by vrezhMik**
I have done some small changes in input, category icons and assets. Changed their transitions, border colors. Changed maximum text length from 200 to 400 for textarea.

### CODE REVIEW

File: src/viewer/components/EditBoard.vue [hunk 1]
```diff
@@ -80,4 +84,24 @@ watch(Store.board_to_edit, () => {
   font-size: 20px;
   line-height: 25px;
 }
+
+.transition {
```
Comment (HaykAvagyan1) [inline src/viewer/components/EditBoard.vue:88]: You can put this transition in a global `<style>` tag at the root so you don't have to add .transition and all the .transition.v-enter or .v-leave code to each component that needs it.

**Review by HaykAvagyan1 — CHANGES_REQUESTED** [review]
Looks good, I left two small comments to address and it's ready to merge! Also I feel like these two commits could be squashed and you can add a small description describing the changes you've done.

File: src/viewer/components/EditBoard.vue [hunk 2]
```diff
@@ -80,4 +84,24 @@ watch(Store.board_to_edit, () => {
...
+}
+.transition.v-leave-from {
+  opacity: 1;
+}
+.transition.v-leave-to {
+  opacity: 0;
+}
+
+.active_frame {
+  /* padding-top: 11px; */
```
Comment (HaykAvagyan1) [inline src/viewer/components/EditBoard.vue:105]: Remove this comment if it's not needed and the tag.

File: src/viewer/components/EditBoard.vue (hunk 1 above)
Comment (vrezhMik) [reply src/viewer/components/EditBoard.vue:88]: That would be great. But i have another styling with that class which applies not only opacity.

File: src/viewer/components/SaveButton.vue [hunk 3]
```diff
@@ -13,12 +13,15 @@
...
 import Store from "@/Store.vue";
-import { ref, watch } from "vue";
+import { onMounted, ref, watch } from "vue";
 let left = ref(0);
 function save() {
   Store.saved_status.value = !Store.saved_status.value;
 }
-
+onMounted(() => {
+  if (Store.saved_status.value) left.value = 35;
```
[the remark this answers is not in the document]
Comment (vrezhMik) [reply src/viewer/components/SaveButton.vue:22]: done

File: src/viewer/components/EditBoard.vue (hunk 1 above)
Comment (HaykAvagyan1) [reply src/viewer/components/EditBoard.vue:88]: Okay, cool.

**Review by HaykAvagyan1 — APPROVED** [review]
Go ahead and merge.

**Pull request merged**